src/asciidoc/docs/_includes/misc.adoc

```diff
@@ -0,0 +1,17 @@
+= Miscellaneous
+
+////
```

Comment:
Another topic to add to our standards discussion. I'm not sure how useful this Usage information will be for the includes vs. the maintenance it creates. For example, the path will vary based on where the include is used. The syntax for including a page is standard Asciidoc, so we're not providing any information that isn't already covered by those references. Some include files are not written in Asciidoc (e.g. code examples) so this pattern won't work.

Reply:
This comment block is very useful for me. I can just copy and paste a single include line, instead of remembering the syntax + copying and pasting the filename + copying and pasting the tag name. This is more proactive than waiting for the build to throw an error to find out the file/tag name is misspelled or a colon is missing.

The comment block is updated, only if includes are added to / removed from that file, so the file is already being updated at that point anyway.

Reply:
If it's a time-saver for you, let's give it a try. Just note that the include path could be different if the place you're including it is different relative to the includes folder (e.g., a /reference folder page or a subfolder page).